srtcore/group.h

```diff
@@ -162,23 +162,8 @@ class CUDTGroup
         {
             m_Group.erase(f);
 
-            // Reset sequence numbers on a dead group so that they are
-            // initialized anew with the new alive connection within
-            // the group.
-            // XXX The problem is that this should be done after the
-            // socket is considered DISCONNECTED, not when it's being
-            // closed. After being disconnected, the sequence numbers
-            // are no longer valid, and will be reinitialized when the
-            // socket is connected again. This may stay as is for now
-            // as in SRT it's not predicted to do anything with the socket
-            // that was disconnected other than immediately closing it.
             if (m_Group.empty())
             {
-                // When the group is empty, there's no danger that this
-                // number will collide with any ISN provided by a socket.
-                // Also since now every socket will derive this ISN.
-                m_iLastSchedSeqNo = generateISN();
-                resetInitialRxSequence();
```

Comment:
To fix the issue, it should keep previous group-wise sequences even if the group is empty.